src/stan_math_backend/Stan_math_code_gen.ml

```diff
@@ -44,6 +44,7 @@ let pp_templates ~defaults ppf templates =
   match templates with
   | [] -> ()
   | _ ->
+      (* TODO collate requires into one require_all *)
```

Comment:
This is a note to remember to come back and do something like what @SteveBronder was working on in #1107. I took a slightly different approach in this PR, but the same method of combining them into one `require_all` should work. My first priority was to get it compiling as-is, however.